packages/regl-worldview/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "regl-worldview",
-	"version": "0.1.2",
+	"version": "0.1.4",
```

Comment:
Should we just do this version bump separately outside of this PR?

Reply:
probably, since it should be 0.1.4 in master

Reply:
Done in #190